### PR TITLE
Web: Add disabled state to RadioGroup and add new icon

### PR DESCRIPTION
**Pull request by kimlisa**
part of https://github.com/gravitational/teleport.e/issues/1899

- adds new icon
- add disabled state to RadioGroup option

Screenshot:

image

image

image

### CODE REVIEW

**Comment by kimlisa** [comment]
tagging @gzdunek for the radio group addition

File: web/packages/design/src/RadioGroup/RadioGroup.tsx [hunk 1]
```diff
@@ -70,6 +71,7 @@ interface RadioProps {
   checked: boolean;
   autoFocus?: boolean;
   onChange?: (value: string) => void;
+  disabled?: boolean;
```
Comment (gzdunek) [inline web/packages/design/src/RadioGroup/RadioGroup.tsx:74]: This is not needed, we only use the value from `RadioObjectOption`.

File: web/packages/design/src/RadioGroup/RadioGroup.tsx [hunk 2]
```diff
@@ -94,14 +99,19 @@ function Radio(props: RadioProps) {
           margin: 0 ${props => props.theme.space[2]}px 0 0;
           accent-color: ${props => props.theme.colors.brand};
           cursor: inherit;
+
+          &:disabled {
+            cursor: not-allowed;
+          }
```
Comment (gzdunek) [inline web/packages/design/src/RadioGroup/RadioGroup.tsx:105]: This is also not needed, we set cursor style on the parent element.

**Review by gzdunek — APPROVED** [review]
With minor comments.

**Review by avatus — APPROVED** [review]

**Pull request merged**